### DISCUSSION TITLE
Client plugins not working

### DISCUSSION THREAD

**Discussion by KwintenCouton1** [moyix/fauxpilot#130] [Q&A] [answered]
Hi, I am getting a JSON error using the VS Code GitHub Copilot plugin with overwritten settings:
`    "github.copilot.advanced": {
        "debug.overrideEngine": "codegen",
        "debug.testOverrideProxyUrl": "http://localhost:5000",
        "debug.overrideProxyUrl": "http://localhost:5000"
    },`

The error shows up in the Github Copilot output window as 
[ERROR] [streamChoices] [2022-12-14T19:53:52.379Z] Error parsing JSON stream data,data: \<what seems to be valid JSON Data>
and then a few more times without any data
I have also tried the GitLab Workflow plugin, but that doesn't even show anything in its output window.
`    "gitlab.aiassist.enabled": true,
    "gitlab.aiassist.server": "http://localhost:5000",
`

As far as I can tell, the server side works fine; I can get a response through Swagger and I see changes in the docker logs when i type in VS Code. 
I am using the codegen-6B-multi model on an RTX 4080 graphics card.
Let me know if I need to provide any other details.
Many thanks!

**Comment by dewacandra4** [comment, marked as answer]
I also encountered the same error, have you found a solution? @KwintenCouton1

**Reply by dewacandra4** [reply]
Found the solution by installing this extension https://marketplace.visualstudio.com/items?itemName=Venthe.fauxpilot
